File: Fundamentos de Desenvolvimento com C#/AT/EX08/Funcionario.cs
namespace EX08 {
    internal class Funcionario {
        public string Nome { get; set; }
        public string Cargo { get; set; }
        public double SalarioBase { get; set; }

        public Funcionario(string nome, string cargo, double salarioBase) {
            Nome = nome;
            Cargo = cargo;
            SalarioBase = salarioBase;
        }

        public virtual double CalcularSalario() {
            return SalarioBase;
        }
    }
}

File: Fundamentos de Desenvolvimento com Java/AT/src/EX08/Funcionario.java
package EX08;

import java.text.DecimalFormat;

public class Funcionario {
    protected String nome;
    protected double salarioBase;

    // Construtor
    public Funcionario(String nome, double salarioBase) {
        this.nome = nome;
        this.salarioBase = salarioBase;
    }

    public double calcularSalario() {
        return salarioBase;
    }

    public void exibirDados() {
        DecimalFormat df = new DecimalFormat("#,###.00");
        System.out.println("Nome: " + nome);
        System.out.println("Salário final: R$ " + df.format(calcularSalario()));
    }
}
